### PR TITLE
13.3 dashboard view

### PR DESCRIPTION
**Pull request by stoeffu**

### CODE REVIEW

**Comment by raeffu** [comment]
Can this branch be merged? Does the JPA shit work?

File: src/main/java/ch/bfh/healvis/ui/controller/DashboardController.java [hunk 1]
```diff
@@ -12,37 +11,54 @@
...
-        ArrayList<Patient> healthVisitorPatients = new ArrayList<>();
-        ArrayList<Appointment> passedAppointments = new ArrayList<>();
-        ArrayList<Appointment> futureAppointments = new ArrayList<>();
-        
-        // iterate through appointments, decide if it's a past or future appointment
-        for (Appointment appointment : appointments) {
+        user = ((HealVisUI) UI.getCurrent()).getCurrentUser();
+
+        // TODO: is this a good way to handle null pointer returned by HealVisUI?
+        if (user == null) {
```
Comment (dave2sidefyn) [inline src/main/java/ch/bfh/healvis/ui/controller/DashboardController.java]: Vorerst find ich dies so okay. :) Gibt später eventuell noch bessere Möglichkeiten

File: src/main/java/ch/bfh/healvis/ui/controller/DashboardController.java [hunk 2]
```diff
@@ -12,37 +11,54 @@
...
+            return;
+        }
```
Comment (dave2sidefyn) [inline src/main/java/ch/bfh/healvis/ui/controller/DashboardController.java]: It is not the best way but okay at the moment. We will go through de security questions in a later timeperiod

File: src/main/java/ch/bfh/healvis/ui/view/DashboardView.java [hunk 3]
```diff
@@ -1,50 +1,51 @@
...
-        patient2.setBirthday(new Date());
-        testData.add(patient2);
-        addComponent(new PatientList(testData));
-        addComponent(new AppointmentList("Passed Appointments", passedAppointments));
-        addComponent(new AppointmentList("Future Appointments", futureAppointments));
-    }
-    
-    public void update () {
+        Label welcomeMessage = new Label("Welcome, " + controller.getUserName() + "!");
+        welcomeMessage.setStyleName("h2");
```
Comment (dave2sidefyn) [inline src/main/java/ch/bfh/healvis/ui/view/DashboardView.java]: Maybe we should use constants?

**Pull request merged**